docs/develop/external-calls.md

```diff
@@ -1,28 +1,29 @@
 ---
 title: 'External API call support in Office Scripts'
 description: 'Support and guidance for making external API calls in an Office Script.'
-ms.date: 09/24/2020
+ms.date: 12/28/2020
 localization_priority: Normal
 ---
 
 # External API call support in Office Scripts
 
-The Office Scripts platform doesn't support calls to [external APIs](https://developer.mozilla.org/docs/Web/API). However, these calls can be run under the right circumstances. External calls can be only be made through the Excel client, not through Power Automate [under normal circumstances](#external-calls-from-power-automate).
+Script authors shouldn't expect consistent behavior when using [external APIs](https://developer.mozilla.org/docs/Web/API) during the platform's preview phase. This is due how the JavaScript runtime manages interacting with the workbook. The script may end before the API call completes (or its `Promise` is fully resolved). As such, do not rely on external APIs for critical script scenarios.
```

Comment:
re: "may end before its Promise is fully resolve" - the script ends when the promise resulting from main is resolved (do we want to be this specific? maybe), and is equally true for external and office js calls (though sync hides that in the officejs case). 

for external apis, the user needs to mark their main as async and await calls, it'll work as expected, eg:

async function main(workbook: ExcelScript.Workbook) {
    workbook.getActiveCell().setValue("Starting");
    const r = await fetch('https://jsonplaceholder.typicode.com/todos/1');
    const j = await r.json()
    workbook.getActiveWorksheet().getRange("A1").setValue(JSON.stringify(j));
    workbook.getActiveCell().setValue("Done");
}

However, the below would have non-deterministic behavior - the Done is likely to happen before the fetch completes, and main would exit and therefore the .then() wouldn't process.

function main(workbook: ExcelScript.Workbook) {
    workbook.getActiveCell().setValue("Starting");
fetch('https://jsonplaceholder.typicode.com/todos/1').then(r => r.json()).then(j =>  workbook.getActiveWorksheet().getRange("A1").setValue(JSON.stringify(j)));
    workbook.getActiveCell().setValue("Done");
}